src/components/AddToDoInput.tsx

```diff
@@ -1,36 +1,35 @@
 import * as React from 'react';
 import { Dispatch, SetStateAction, useState } from 'react'
 import { ToDo } from '../App';
+import { useContext } from 'react';
+import { TodoContext, ToDoContextInterface } from '../context/ToDoContext';
+
 
 
-export interface AddToDoInputProps {
-    setToDoList: Dispatch<SetStateAction<ToDo[]>>
-    toDoList: ToDo[]
-}
 
-export function AddToDoInput(props: AddToDoInputProps) {
+export function AddToDoInput() {
 
-    const [title, setTitle] = useState("")
-    const [description, setDescription] = useState("")
+    // const [title, setTitle] = useState("")
+    // const [description, setDescription] = useState("")
 
-    const submitToDo = (e: any) => {
-        e.preventDefault();
-        console.log([...props.toDoList, { id: props.toDoList.length + 1, title: title, description: description }])
-        return props.setToDoList([...props.toDoList, { id: props.toDoList.length + 1, title: title, description: description }])
-    }
+    // const submitToDo = (e: any) => {
+    //     e.preventDefault();
+    //     return props.setToDoList([...props.toDoList, { id: props.toDoList.length + 1, title: title, description: description }])
+    // }
+    const toDoContext: ToDoContextInterface = useContext(TodoContext)
```

Comment:
You could distructure the required properties 
```suggestion
    const { handleSubmitToDo, title, setTitle, setDescription } : ToDoContextInterface = useContext(TodoContext)
```